offClick.js

```diff
@@ -10,7 +10,24 @@ angular.module('offClick', [])
         if (!target || !elms) return false;
         var elmsLen = elms.length;
         for (var i = 0; i < elmsLen; ++i)
-        if (elms[i].contains(target)) return true;
+        if (elms[i].contains(target)) {
```

Comment:
Totally, you are right. This was exactly the line I wanted to replace.

Sorry about that, I was in a hurry yesterday. Should be fixed now.